src/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Button.css';
+
+function Button(props) {
+    return (
+        <button className={`button ${props.modifierClassName}`} {...props}>
```

Comment:
I would suggest switching `modifierClassName` back to `className` and then ensuring you're not passing the prop down to `button` twice, like so.

```suggestion
function Button({ className, ...props }) {
    return (
        <button className={`button ${props.className}`} {...props}>
```

(I think that will work 🤞 )

Reply:
You'll also have to update some of the code below to match.